<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialsWindow</class>
 <widget class="QWidget" name="MaterialsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1257</width>
    <height>652</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1257</width>
    <height>652</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Справочник материалов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/images/logo/images/icon.ico</normaloff>:/images/logo/images/icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>300</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>300</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>620</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>42</y>
     <width>20</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>620</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>300</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1231</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
    </property>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>236</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Фильтр</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>500</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="materialsTreeWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>41</y>
     <width>230</width>
     <height>571</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>341</y>
     <width>481</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Соответствие работам</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="materialWorkAppropriatenessTableWidget">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Соответствие</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>341</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Дополнительные параметры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="materialsExtraOptionsTableWidget">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Значение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ед. изм</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="materialsTableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>42</y>
     <width>981</width>
     <height>251</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>№</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ед. изм</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Цена</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Мин. кол-во</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Вес</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Коэф. отх</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
